modules/ROOT/pages/bat-bdd-reference.adoc

```diff
@@ -84,6 +84,70 @@ describe `Demo scenario` in [
+b. Select Shared Secret on the left side of the screen.
+c. In the *Type* field, select *Symmetric Key*.
+d. In the *Key* field, paste your license key encoded as a Base64 string.
+e. Paste the same Base64 string into the *Confirm Key* field.
+. Copy the name of the new shared secret.
+. At a command prompt, run the `bat grant` command, specifying an alias for the shared secret. When you run this command, the BAT CLI creates a section named `secrets` in your test suite's `bat.yaml` file, if the section does not already exist. In that section, the BAT CLI adds these indented lines:
++
+----
+  alias:
+    secretId: "secret-ID"
+----
++
+alias:: This is the alias that you specified in the `bat grant` command.
+secret-ID:: This is the ID of the secret within Anypoint Secrets Manager. This ID does not appear in ASM, so there is no way for someone looking in your `bat.yaml` file to associate the ID with any particular secret. The BAT CLI uses this ID to look up the secret that you associated with the alias.
+. Use the alias in a test by storing it in a variable, as in this example:
++
+----
+import * from bat::BDD
+import * from bat::Assertions
+
+var mySecret = String = secret(secretAlias='myAlias') default 'notFound'
+
+---
+suite(parts="Simple Suite") in [
+  it must 'answer 200' in [
+    GET '$(config.url)?token=$(mySecret)' with {} assert [
+      $.response.status mustEqual 200
+    ] execute
+  ]
+]
+----
++
+`mySecret`:: The variable in which the secret is stored in this example. The variable is declared as a String.
+`secret(secretAlias='myAlias')`:: The `secret()` function requires the input `secretAlias='name-of-alias')`, where `name-of-alias` is the alias of the shared secret that you want to use.
+`default 'notFound'`:: The message to show if a shared secret with the specified alias does not exist in Anypoint Secrets Manager.
+`$(mySecret)`:: The use of the variable `mySecret` to supply the authentication token for the endpoint at the URL that is defined in the config file that was specified when the `bat` command to run the test suite was entered. Suppose that the config file was this:
++
+.The example config file dev-environment.dwl
+----
+{
+  url: "https://dev.yourcompany.com/shoppingCart"
+}
+----
++
+Then, the command to run this test suite, when the command is run in the directory in which the `bat.yaml` file for the test suite is located and the default profile is used, is this:
```

Comment:
```suggestion
In this case, the command to run this test suite, when the command is run in the directory in which the `bat.yaml` file for the test suite is located and the default profile is used, is this:
```
Is that long parenthetical in the middle of this sentence really necessary to audience understanding?